tests/regression/77-lin2vareq/05-vars_equality.c

```diff
@@ -0,0 +1,25 @@
+//SKIP PARAM: --set ana.activated[+] lin2vareq --set ana.int.enums false --set ana.int.interval false --set ana.int.interval_set false --set ana.int.congruence false
+
+#include <stdio.h>
+
+int main() {
+    int x1 = 5, x2 = 10, x3 = 15, x4, x5, x6, x7, x8, x9, x10, x11, x12;
+
+    x4 = 3 * x2 + 5;
+    x5 = 3 * x3 + 15;
+    x6 = x3 + 3;
+    x7 = x3 + 2;
+    x8 = 7 * x3 + 15;
+    x9 = 0;
+    x10 = 2 * x9 + 2;
+    x11 = 2 * x1 - 3;
+    x12 = 4 * x1 - 5;
```

Comment:
These also seem to just compute with known values that `def_exc` can handle, no?